polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -1102,11 +1103,20 @@ llvm::Optional<sycl::SYCLMethodOpInterface> MLIRScanner::createSYCLMethodOp(
   LLVM_DEBUG(llvm::dbgs() << "Inserting operation " << OptOpName
```

Comment:
[nit]: Adjust the msg because the code after this line might not insert the SYUCL method. Propose:

"Inserting operation" -> "Attempting to insert operation"

Reply:
Done!